core/src/main/scala/org/apache/spark/SparkContext.scala

```diff
@@ -2183,6 +2183,13 @@ object SparkContext extends Logging {
   }
 
   /**
+   * Return SparkContext
+   */
+  def get(): SparkContext = {
```

Comment:
We could potentially just drop a message if it cannot be sent and log a warning?